README.md

```diff
@@ -14,7 +14,6 @@ simply run:
 
 ```sh
 npm install
-grunt
```

Comment:
@josephfrazier we should also take into account that the `devDependencies` are not retrieved when the package is installed as a dependency of another one, and so I'm not entirely sure that defining `prepublish`/`prepare` will be enough to get the the dist files correctly built in this scenario.